frontend/src/metabase/css/dashboard.module.css

```diff
@@ -34,13 +34,6 @@
   fill: currentColor;
 }
 
-.Dashboard.DashboardNight :global(.border-top),
```

Comment:
This part is a little tricky to verify.

I checked all the components that use `CS.borderTop`, `CS.borderLeft`, `CS.borderDown`, `CS.borderRight`

And all of them aren't rendered inside the dashboard.

We can rule out:
- all of the admin components.
- Any dashboard filter because it will rendered in a portal outside `.Dashboard.DashboardNight` anyway
- Any other setting components, since dashboards couldn't be in night mode when editing. They can only be set to night mode when 1) in fullscreen 2) static embedding (and maybe public sharing too)